package.json

```
@@ -129,7 +124,9 @@
     "build:prep:publish": "ts-node build-utils/setupPackage.ts",
     "postbuild:prep:publish": "echo 'done building for publishing'",
     "test": "jest --watchAll",
-    "test:prod": "jest",
+    "test:prod": "jest --testPathIgnorePatterns storyshots",
+    "test:normal": "jest --testPathIgnorePatterns storyshots",
```

```suggestion
    "test:prod": "jest --testPathIgnorePatterns storyshots",
```
I think "test:normal" is not necessary

---

So there is way to do the snapshot on a story-by-story basis. The `puppeteerTest` section of [this](https://storybook.js.org/addons/@storybook/addon-storyshots-puppeteer). Then it would allow us to interact with the  component before we take the screenshot.

---

Gotcha. Can't wait to see when that can also be implemented 🥳 . This is a good testing implementation for a UI library, really easy to see what has changed when having a picture snapshot of it on the PR. 💯